docs/x509/reference.rst

```diff
@@ -499,6 +499,16 @@ X.509 Certificate Object
         :return bytes: The data that can be written to a file or sent
             over the network to be verified by clients.
 
+    .. method:: check_issued(issuer)
+
+        .. versionadded:: 3.1
+
+        This function checks if certificate subject was issued using CA certificate issuer.
```

Comment:
Should the comment above starting with "To validate the signature on a certificate ..." be updated?

Reply:
Removed it and added a reference to the new validation function.